packages/oranger/oranger.2.0.2/files/compile_ranger.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -x
+
+tar xzf 0.9.11.tar.gz
+
+cd ranger-0.9.11/cpp_version
+mkdir build
+cd build
+cmake ../
+make
+cp ranger `opam config var bin`/ml_rf_ranger
```

Comment:
I think it would be better if `%{bin}%` was passed as parameter instead

Reply:
done